event_manager.py

```diff
@@ -105,6 +106,18 @@ def listen():
     return None
 
 
+@app.route('/leaderboard', methods=['POST'])
+def slash_command():
```

Comment:
This should be renamed to `show_leaderboard` or something similar.

Reply:
Should this match the slash command exactly? The slash command is just /leaderboard currently.

Reply:
The route is fine, I just want the name of the method changed.

Reply:
```suggestion
def show_leaderboard():
```